### DISCUSSION TITLE
Is it possible to deploy as an AWS Lambda Function?

### DISCUSSION THREAD

**Discussion by CptDaimox** [gotenberg/gotenberg#770] [Q&A]
Hello!

I was wondering if it is somehow possible to deploy/host Gotenberg as an AWS Lambda Function and to trigger it with an API Gateway Event.
My initial ideas were to fork the project and modify the code so it is compatible with Lambda, and build the image accordingly. However, this would take too much time, as my knowledge of Go and Docker is not yet advanced enough 😅.

Thank you in advance! 🫶

**Comment by gulien** [comment]
Hello @CptDaimox,

See #540. IMO, while the code implementation could be done quite quickly, there are a lot of caveats:

* How to adapt de Docker image?
* ALB + GW support
* What are the limitations?
